gulpfile.js

```diff
@@ -211,7 +211,7 @@ export async function buildTestAppJs() {
   } else {
     testAppBuildResult = await esbuild.build({
       entryPoints: ['./test-app/bundles/main.js'],
-      target: ['esnext', 'chrome95', 'firefox93', 'safari15', 'edge95'],
+      target: ['esnext', 'chrome95', 'firefox93', 'safari13', 'edge95'],
```

Comment:
I'm not sure what to do with the CSS bundler

Reply:
It would be cool if you could also make sure that Autoprefixer is configured to respect Safari >=13. Otherwise we need to make a second ticket for that.

Reply:
I hope I did it right, I knew there was something with the autoprefixer but I'm not sure how to test it.

Reply:
Yes, it's right. You can always check at: https://browserslist.dev/